build.gradle

```diff
@@ -2,35 +2,36 @@
 buildscript {
 
     ext {
-        kotlin_version = "1.4.32"
+        kotlinVersion = '1.5.21'
         retrofitVersion = '2.9.0'
-        navigationVersion = "2.3.4"
+        navigationVersion = '2.3.4'
```

Comment:
We have two navigationVersion declarations.

Reply:
Good catch!